libs/ui-watt/src/lib/components/slider/watt-slider.stories.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { moduleMetadata, Story, Meta } from '@storybook/angular';
+
+import { WattSliderComponent } from './watt-slider.component';
+import { WattSliderModule } from './watt-slider.module';
+
+export default {
+  title: 'Components/Slider',
+  component: WattSliderComponent,
+  decorators: [
+    moduleMetadata({
+      imports: [WattSliderModule],
+    }),
+  ],
+} as Meta<WattSliderComponent>;
+
+export const Overview: Story<WattSliderComponent> = (args) => ({
+  props: args,
+});
+
+Overview.args = {
+  value: [0, 100],
+};
+
+export const TimePicker: Story<WattSliderComponent> = (args) => {
+  return {
+    props: {
+      ...args,
+      format: (x: number) => {
+        const hours = x === 96 ? '23' : `${Math.floor(x / 4)}`;
+        const minutes = x === 96 ? '59' : `${(x * 15) % 60}`;
+        return `${hours.padStart(2, '0')}:${minutes.padStart(2, '0')}`;
+      },
+    },
+    template: `
+      <watt-slider [(value)]="value"></watt-slider>
+      <p>{{format(value[0])}} - {{format(value[1])}}</p>
+    `,
+  };
+};
+
+TimePicker.args = {
+  max: 96,
+  value: [25, 75],
+};
```

Comment:
praise: this looks quite nice. Good job 💪

question: do we want to show a time picker-looking functionality as part of the slider stories? We know that the slider component is going to be integrated with the time picker component, so we should probably leave keep this story when we do that. Another point if favor of that is that including the functionality here might lead to the wrong impression that this is how a time picker component should be implemented in the apps.

What do you think? Also @frontend-specialisten

Reply:
My initial idea was to remove this story once the time picker is in place, so sounds like it aligns with your thoughts as well? I created this story primarily for testing how it would work for selecting a time range. I could also just remove it right away if that is better, I am fine either way 🙂

Reply:
I think it's totally fine as @ManBearTM has planned. Currently we don't have any usecases for the slider alone, so I don't see any problems having this until the timepicker has been finished :)

Reply:
Let's leave it as is. We'll move it as part of implementing the timepicker.